BookReader/plugins/plugin.search.js

```diff
@@ -41,6 +41,12 @@ BookReader.prototype.init = (function (super_) {
                 goToFirstResult: true
             });
         }
+
+        var br = this;
+        $(document).on('BookReader:navToggled', function() {
+            var nextVisibleState = br.navigationIsVisible() ? 'visible' : 'hidden';
```

Comment:
QUESTION: Wait, wouldn't this be the opposite? If this is a toggle and the navigation is currently visible, then wouldn't the next visible state be "hidden"?

Reply:
thanks for catching, naming has been updated.